VaticanBAV/et/BAVet117.xml

```diff
@@ -0,0 +1,1158 @@
+                            </explicit>
+                            <listBibl type="editions">
+                               <bibl>
+                                  <ptr target="bm:Budge1900Miracles"/>
+                                  <citedRange unit="page">40-41</citedRange>
+                               </bibl>
+                            </listBibl>
+                            <listBibl type="translation" xml:lang="en">
+                               <bibl>
+                                  <ptr target="bm:Budge1900Miracles"/>
+                                  <citedRange unit="page">71-72</citedRange>
+                               </bibl>
+                            </listBibl>
+                          </msItem>
+
+                          <msItem xml:id="ms_i1.5.2">
+                            <locus from="16v" to="16v"/>
+                            <title>Salām</title>
+                            <incipit xml:lang="gez">ሰላም፡ ዕብል፡ አስይፍት፡ ወኲያንው፡ <gap reason="omitted" extent="unknown" resp="PRS4805Grebaut PRS9530Tisseran"/></incipit>
+                          </msItem>
+
+                        </msItem>
+
+                        <msItem xml:id="ms_i1.6">
+                          <locus from="16v" to="17v"/>
+
+                          <msItem xml:id="ms_i1.6.1">
+                            <locus from="16v" to="17v"/>
+                            <title type="complete" ref="LIT3613Miracle"/>
+                            <incipit xml:lang="gez">
+                              ተአምሪሃ፡ <gap reason="omitted" extent="unknown" resp="PRS4805Grebaut PRS9530Tisseran"/> ወሀሎ፡ ፩ብእሲ፡ ዘስሙ፡ አንስጣስዮስ፡ ፈራሄ፡ እግዚአብሔር፡ ወያፈቅራ፡ ለእግዝእትነ፡
+                            </incipit>
+                            <explicit xml:lang="gez">
+                              <choice><sic>ወውእተ፡ ጊዚ፡</sic></choice> አዕረፈ፡ ወተመጠ<choice><corr>ወ</corr></choice>ት፡ ነፍሶ፡ ንጽሕተ፡ እግዝእትነ፡ ኦእግዝእትየ፡ ጸሎትኪ፡ <gap reason="omitted" extent="unknown" resp="PRS4805Grebaut PRS9530Tisseran"/>
+                            </explicit>
+                            <listBibl type="editions">
+                               <bibl>
+                                  <ptr target="bm:Budge1900Miracles"/>
+                                  <citedRange unit="page">41-42</citedRange>
+                               </bibl>
+                            </listBibl>
+                            <listBibl type="translation" xml:lang="en">
+                               <bibl>
+                                  <ptr target="bm:Budge1900Miracles"/>
+                                  <citedRange unit="page">73-74</citedRange>
+                               </bibl>
+                            </listBibl>
+                          </msItem>
+
+                          <msItem xml:id="ms_i1.6.2">
+                            <locus from="17v" to="17v"/>
+                            <title>Salām</title>
```

Comment:
Aha, I hadn't realized that the salāms often found with the Miracles of Mary corresponded to another work.

For the ones that don't correspond though, I wouldn't imagine that it would be worthwhile in most of the cases? I am sure there is an inexhaustible supply of unique salāms in such works.

1.7.2 though (ይትባረክ፡ ጽጌኪ፡ ማርያም፡ -- which I guess is not found in ʾAkko-nu bəʾsi?) is actually from Nags za-māḥǝlet (or at least is used there), but I would guess that that is an exceptional case (since it's not at all related to the content of the miracle).

Reply:
The text identification issue should be opened in any case - its result can always be that no new ID should be created. We do however create IDs for such salāms, because this helps identifying them when they are found elsewhere, which does happen.
This manuscript is interesting, because it does not contain the classical ʾAkko-nu bəʾsi collection.